inst/templates/widget_r.txt

```diff
@@ -0,0 +1,52 @@
+#' <Add Title...>
+#'
+#' <Add Description>
+#'
+#' @import htmlwidgets
+#'
+#' @export
+%s <- function(message, width = NULL, height = NULL, elementId = NULL) {
+
+  # forward options using x
+  x = list(
+    message = message
+  )
+
+  # create widget
+  htmlwidgets::createWidget(
+    name = '%s',
+    x = x,
+    width = width,
+    height = height,
+    package = '%s',
+    elementId = elementId
+  )
+}
+
+#' Shiny bindings for %s
+#'
+#' Output and render functions for using %s within Shiny
+#' applications and interactive Rmd documents.
+#'
+#' @param outputId output variable to read from
+#' @param width,height Must be a valid CSS unit (like \\code{'100\\%%'},
```

Comment:
Oh you know what, I think I needed the extra `\` since I had the template inline earlier. Now that I have moved it into a file, I can just stick with a single `\` probably.